packages/@aws-cdk/aws-apigateway/test/integrations/stepfunctions.test.ts

```diff
@@ -0,0 +1,275 @@
+import '@aws-cdk/assert-internal/jest';
+import * as sfn from '@aws-cdk/aws-stepfunctions';
+import { StateMachine, StateMachineType } from '@aws-cdk/aws-stepfunctions';
+import * as cdk from '@aws-cdk/core';
+import * as apigw from '../../lib';
+
+function givenSetup() {
```

Comment:
nit: Nice to move all these helper functions to the bottom of the file, rather than at the top.